jenkins-plugin/src/test/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepTest.java

```diff
@@ -68,7 +68,7 @@ public void setup() throws Exception {
         JavaGitContainer slaveContainer = slaveRule.get();
 
         DumbSlave agent = new DumbSlave("remote", "", "/home/test/slave", "1", Node.Mode.NORMAL, "",
```

Comment:
```suggestion
        SystemCredentialsProvider.getInstance().getDomainCredentialsMap().put(Domain.global(), Collections.singletonList(new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, "test", null, "test", "test")));
        DumbSlave agent = new DumbSlave("remote", "", "/home/test/slave", "1", Node.Mode.NORMAL, "",
```

plus fix imports. The `SSHLauncher` ctor changed from raw secrets to `credentials`.